legal-api/src/legal_api/models/business.py

```diff
@@ -341,6 +341,7 @@ def json(self):
             'nextAnnualReport': LegislationDatetime.as_legislation_timezone_from_date(
                 self.next_anniversary
             ).astimezone(timezone.utc).isoformat(),
+            'associationType': self.association_type
```

Comment:
I think this is applicable only for coops. So not sure whether this should be in the response only if self.legal_Type=='CP'?

Reply:
Its applicable for `SP` as well